### PR TITLE
Logging in on the website

### PR DESCRIPTION
**Pull request by MlulekiN**

### CODE REVIEW

File: properties/Configuration.properties [hunk 1]
```diff
@@ -1,4 +1,4 @@
-baseUrl=https://dev-aluvegh.co.za/index.html
+baseUrl=https://aluveapp-qa.co.za/admin/
```
Comment (benedictnkosi) [inline properties/Configuration.properties]: create new variable for this

File: src/test/java/pages/LoginPage.java [hunk 2]
```diff
@@ -11,21 +11,27 @@
...
     WebDriver driver;
-    By password = By.id("secret");
-    By loginButton = By.xpath("//button[@type=\"submit\"]");
-    By loginPageHeader = By.xpath("//h2[@class='login-heading']");
-    By loggedInHeader = By.xpath("//h1[text()='Guesthouse Management']");
+    By email = By.xpath("//input[@placeholder='Email']");
+    By password = By.xpath("//input[@placeholder='Password']");
+    By loginButton = By.xpath("//input[@type=\"submit\"]");
+    By loginPageHeader = By.xpath("//h1[text()='Log In Page']");
+    By loggedInHeader = By.xpath("//div[@class='profile-top']/descendant::h5[text()='Calendar']");
```
Comment (benedictnkosi) [inline src/test/java/pages/LoginPage.java]: this element is not on the loginPage

File: Feature/Login.feature [hunk 3]
```diff
@@ -0,0 +1,12 @@
+Feature: Login Action
+
+  Background:
+    Given User is on Login Page
+
+  Scenario Outline: Successful Login with Valid Credentials
+    When User enters the email '<email>'
+    When User logs in with "<password>"
+    Then User is redirected to the Dashboard page
```
Comment (benedictnkosi) [inline Feature/Login.feature]: to the calendar page

**Review by benedictnkosi — CHANGES_REQUESTED** [review]

File: properties/Configuration.properties (hunk 1 above)
Comment (MlulekiN) [reply properties/Configuration.properties]: Done

File: src/test/java/pages/LoginPage.java (hunk 2 above)
Comment (MlulekiN) [reply src/test/java/pages/LoginPage.java]: Done

File: Feature/Login.feature (hunk 3 above)
Comment (MlulekiN) [reply Feature/Login.feature]: Done

**Review by benedictnkosi — APPROVED** [review]

**Pull request merged**